hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/api/StringMatcher.kt

```diff
@@ -68,13 +71,14 @@ data class StringMatcher(
   
   IS and IS_NOT require an exact match (wildcards ? and * will not work).
   
-  CONTAINS checks for a partial match and respects wildcards ? (single character) and * (zero to many characters).
+  For IS and CONTAINS, if the attribute contains free text then the search will also perform a fuzzy (partial) match.
   
-  For IS and CONTAINS, if the attribute contains free text then the search will also perform a fuzzy match.
+  CONTAINS with wildcards ? (single character) and * (zero to many characters) will not perform a fuzzy match.
```

Comment:
Should have clarified this in previous PR to preclude the fuzzy search for CONTAINS with a wildcard.

Didn't bother adding any rules for STARTSWITH, I think it speaks for itself.